.github/workflows/pr_trade_finance_manager-api.yml

```diff
@@ -213,7 +213,7 @@ jobs:
         cd $GITHUB_WORKSPACE
         npm install
         cd e2e-tests/submit-to-trade-finance-manager
-        npx cypress run --config video=false --browser=chrome
+        npx cypress run --config video=true --record --browser=chrome
```

Comment:
the `video=true` flag is not needed anymore. we can change it to `video=false` otherwise the tests will take longer to finish.
not a showstopper though